spec/integration/user_show_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
+RSpec.feature 'User post index page', type: :feature do
+  let!(:user) { User.create!(name: 'Tom', bio: 'Teacher from Mexico', posts_counter: 0) }
+  let!(:post1) { Post.create!(title: 'Hello', text: 'This is my first post', author: user) }
+  let!(:post2) { Post.create!(title: 'Second post', text: 'This is my second post', author: user) }
+  let!(:comment1) { Comment.create!(text: 'hi there', post: post1, author: user) }
+  let!(:comment2) { Comment.create!(text: 'Thanks for sharing', post: post1, author: user) }
+  let!(:like1) { Like.create!(post: post1, author: user) }
+
+  describe 'User posts index page' do
+    before { visit user_posts_path(user) }
+
+    describe 'User profile information' do
+      it 'displays user photo' do
+        expect(page).to have_css("img[src*='#{user.photo}']")
+      end
+
+      it 'displays user name' do
+        expect(page).to have_content(user.name)
+      end
+    end
+
```

Comment:
- [ ] It would be better if we address the comments made by the previous code reviewer, here is a screenshot of what test needs to be added
![Screenshot from 2023-04-19 19-06-07](https://user-images.githubusercontent.com/46388113/233174390-20d11b87-2ef1-47cb-995a-2185ec87950f.png)
![Screenshot from 2023-04-19 19-06-23](https://user-images.githubusercontent.com/46388113/233174404-981f8862-d3a1-422c-bc3a-6f5e76d97746.png)